packages/protocol/contracts/common/linkedlists/IntegerLinkedList.sol

```diff
@@ -0,0 +1,93 @@
+pragma solidity ^0.5.13;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+import "./LinkedList.sol";
+
+/**
+ * @title Maintains a doubly linked list keyed by uint256.
+ * @dev Following the `next` pointers will lead you to the head, rather than the tail.
+ */
+library IntegerLinkedList {
```

Comment:
I don't have an especially strong take either way, I mostly created this to follow the pattern we've done for other type-specific LinkedList / SortedLinkedList libraries.

No idea if the compiler is smart enough to really help though. Fwiw because the library is all internal functions, iiuc the library will be deployed alongside GrandaMento.sol as bytecode that's JUMP'd to and not as a linked library that's delegatecall'd. So while there's probably an extra JUMP in there the gas costs probably aren't exorbitant. Sorta leaning toward keeping it the way it is but am not opposed to changing if there are strong opinions